src/components/logger/index.ts

```diff
@@ -38,20 +38,21 @@ const EventEmitter = getEventEmitter()
 
 /**
  * getStatusColor colorizes different statusCode
- * @param {number} statusCode is the httpStatus to colorize
+ * @param {any} responseCode is the httpStatus to colorize
  * @returns {string} color code based on chalk: Chalk & { supportsColor: ColorSupport };
  */
-export function getStatusColor(statusCode: number) {
-  switch (Math.trunc(statusCode / 100)) {
+export function getStatusColor(responseCode: any) {
```

Comment:
Why is this type `any`?

Reply:
thanks..good catch.  its back to :number